spec/rails_helper.rb

```diff
@@ -41,6 +41,13 @@ def json
       end
     }
   end
+  Shoulda::Matchers.configure do |config|
+    config.integrate do |with|
+      with.test_framework :rspec
+      with.library :rails
+    end
+  end
+  
```

Comment:
why we need this code!?

Reply:
as for model rspec we need to use shoulda_matcher gem for the association and validations spec we need to used shoulda_matcher without this it wasn't working. and when we did bundlw install it got updated into the particular file by itself.